src/odemis/gui/conf/data.py

```diff
@@ -761,6 +776,15 @@
             ("band", {  # from filter
                 "label": "Filter",
             }),
+            # overwrites config
```

Comment:
No need for this comment (as it's the whole point of putting VA info here). However, please move it up, as I'd rather see the (local) VAs from the streams before the (global) axes.